### ISSUE TITLE
It takes many clicks to find out what upload I was thanked for

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#5605]
When I receive a thank notification, it is usually about one of my uploads (the thank usually being sent from Peer Review).

I have unfortunately given checking what picture a thank was for, as it takes time.

If the thanked image was shown directly inside the notifications list, it would be nice.

https://github.com/commons-app/apps-android-commons/assets/99590/c4d7c578-9c04-4492-ae72-90db50997dc3

https://commons.wikimedia.org/w/index.php?title=File:Artefacts_at_Tsukuba_Space_Center_exhibition_hall_36.jpg&oldid=prev&diff=857970978

How the UI could look like:
![IMG_20240305_191408.jpg](https://github.com/commons-app/apps-android-commons/assets/99590/3b45ccae-39fe-4fd0-a6ac-5dde54f58186)

Bonus if the image is tappable, bringing me to its in-app page where I can add metadata to make the picture even more useful.

How the UI is now:
![Screenshot_20240305-190654.png](https://github.com/commons-app/apps-android-commons/assets/99590/abe0823c-0c91-4560-9750-95dd72156b88)

**Comment by neeldoshii** [comment]
Hi @nicolas-raoul, I was testing on prod-debug nominaton for the deletion issue (#5531) on my dummy images, this lead my account to be banned. On beta-debug the nomination to delete the API was resulting as error. 
So I was testing on my dummy contributions nominate to delete.

I understand that on prod its not recommended to upload non-encyclopaediac image and I am really sorry towards it. 

I would like to know further steps to perform unban on the account.

Also, on such cases while working on prod and testing on images how should we do?

**Comment by nicolas-raoul** [comment]
@neeldoshii Would you mind creating a new issue about your block? Please share your Wikimedia username there. Thanks!

**Comment by neeldoshii** [comment]
> @neeldoshii Would you mind creating a new issue about your block? Please share your Wikimedia username there. Thanks!

Issue block over github or on wikimedia? also would like to work on this issue ;)

**Comment by nicolas-raoul** [comment]
If I understand correctly, you were blocked on Wikimedia? Please create a GitHub issue about that, we might be able to help your case.

It seems inconvenient to work on this issue while being blocked, better choose a different one. :-)
